modules/services/window-managers/i3-sway/sway.nix

```diff
@@ -428,6 +441,13 @@ in {
       description = "Sway configuration options.";
     };
 
+    checkConfig = mkOption {
```

Comment:
I'm guessing one would always want to check the configuration? Perhaps could add `visible = false;` or even `internal = true;` to avoid having the option show up in the documentation?